sky/clouds/gcp.py

```diff
@@ -35,8 +35,34 @@
 # This is not a complete list but still useful to check first
```

Comment:
This is *likely* a complete list and is useful ...

Reply:
Good catch! Fixed. Thanks!